collex-loader/load_events.py

```diff
@@ -97,7 +98,13 @@ def get_collection_exercise_uuid(row, api_config):
     print("Dry run: {}".format(dry_run))
 
     api_config = config['api']
-    event_handler = dump_event if dry_run else partial(post_event, user=api_config['user'], password=api_config['password'], url=api_config['post-url'])
+    if dry_run:
+    	event_handler = dump_event 
+    else: 
+	partial(post_event, 
```

Comment:
Think this is broken 👎

Reply:
yeah this wont run for me anymore

Reply:
Hmm ok, I'll take a look and fix when I get home